src/server/project.ts

```diff
@@ -1589,8 +1589,27 @@ namespace ts.server {
             const logError = (message: string) => {
                 (errorLogs || (errorLogs = [])).push(message);
             };
-            const resolvedModule = firstDefined(searchPaths, searchPath =>
+
+            let resolvedModule: any | undefined;
+            if (this.projectService.host.importServicePlugin) {
+                for (const searchPath of searchPaths) {
+                    try {
+                        resolvedModule = await this.projectService.host.importServicePlugin(searchPath, pluginConfigEntry.name);
+                    }
+                    catch (e) {
+                        // TODO: log this?

```

Comment:
You should be able to just use `logError`/`log` to log the exception like we do for `resolveModule` below.